### DISCUSSION TITLE
Workgroup Sync - Week of 8/10

### DISCUSSION THREAD

**Discussion by scovetta** [ossf/wg-identifying-security-threats#2] [General]
I'll be setting up an initial workgroup sync meeting for the week of 8/10. If you're interested in attending, please [e-mail me](mailto:michael.scovetta@microsoft.com?subject=openssf+wg+meeting) and I'll make sure you're invited.

**Comment by DixonDalmeida** [comment]
I have sent an email. Looking forward to the weekly meetings

**Reply by scovetta** [reply]
Thanks @DixonDalmeida - you should have received the invite for this Monday morning (PT).
